react/features/mobile/navigation/functions.js

```diff
@@ -29,3 +35,45 @@ export function screenHeaderCloseButton(goBack: Function) {
             src = { IconClose } />
     );
 }
+
+/**
+ * Determines whether the {@code Prejoin page} is enabled by the app itself
+ * (e.g. Programmatically via the Jitsi Meet SDK for Android and iOS).
+ *
+ * @param {Function|Object} stateful - The redux state or {@link getState}
+ * function.
+ * @returns {boolean} If the {@code Prejoin} is enabled by the app, then
+ * {@code true}; otherwise, {@code false}.
+ */
+export function isPrejoinPageEnabled(stateful: Function | Object) {
+    return getFeatureFlag(stateful, PREJOIN_PAGE_ENABLED, true);
+}
+
+/**
+ * Close icon/text button for lobby screen based on platform.
+ *
+ * @returns {React.Component}
+ */
+export function lobbyScreenHeaderCloseButton() {
```

Comment:
@saghul Created this because I tested with Welcome -> Prejoin -> Lobby. While writing this comment, I realised that from lobby we need to go back to Prejoin if that is available, if not we navigate to Welcome and for SDK sendReadytoClose, right?

Reply:
When in the lobby, if you hangup we should leave completely, not go back to the prejoin page.